ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/fragment/AEMArticleItemFragment.java

```diff
@@ -0,0 +1,177 @@
+package org.cru.godtools.articles.aem.fragment;
+
+import android.net.Uri;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Message;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebResourceResponse;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import org.cru.godtools.article.aem.R;
+import org.cru.godtools.article.aem.R2;
+import org.cru.godtools.articles.aem.db.ArticleRoomDatabase;
+import org.cru.godtools.articles.aem.db.AttachmentRepository;
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.model.Attachment;
+import org.cru.godtools.base.tool.fragment.BaseToolFragment;
+
+import java.lang.ref.WeakReference;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class AEMArticleItemFragment extends BaseToolFragment {
+    public static final String TAG = "AEMArticleItemFragment";
+    private static final String ARTICLE_KEY_TAG = "article_key";
+    private Uri mArticleKey;
+    private List<Attachment> mArticleAttachments;
+    private ArticleRoomDatabase mRDb;
+    private Article mArticle;
+    private static final int MESSAGE_SET_ARTICLE = 101;
+
+    @BindView(R2.id.aem_article_web_view)
+    WebView mAemWebView;
+
+    public static AEMArticleItemFragment newInstance(String tool, Locale locale, String articleKey) {
+        AEMArticleItemFragment fragment = new AEMArticleItemFragment();
+        Bundle args = new Bundle();
+        populateArgs(args, tool, locale);
+        args.putString(ARTICLE_KEY_TAG, articleKey);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    //region LifeCycle
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        Bundle args = getArguments();
+        if (args != null) {
+            mArticleKey = Uri.parse(args.getString(ARTICLE_KEY_TAG));
+        }
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_aem_article_item, container, false);
+        ButterKnife.bind(this, view);
```

Comment:
this is already handled in `onViewCreated` in the base fragment.

Reply:
so, this method can simply be:
```
return inflater.inflate(R.layout.fragment_aem_article_item, container, false);
```

Reply:
OK thanks didn't know that